pkg/merge/aggr_unit.go

```diff
@@ -0,0 +1,26 @@
+//
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package merge
+
+type AggrUnit interface {
+	Merge(values []interface{})
```

Comment:
Emm... How about use Reduce instead of Merge?

Reply:
By the way, why use `[]interface{}` here?

Reply:
> By the way, why using `[]interface{}`?
We can't determine the number of parameters.
For example, for the combination of Count, Max and Min functions, you only need to pass one parameter; However, for Avg function, you need to pass in count and sum parameters before merging